### PR TITLE
Add titles to all pages

### LINKED ISSUE #1882: Page titles
Currently most (all?) Huginn pages have the title "Huginn". This is obviously very undescriptive, and we should go through and make them more descriptive. (And to be clear, when I say "title" I mean the text that's in `<title>` tags.)

### PR DESCRIPTION
**Pull request by strugee**
This should add page titles to all pages linked somewhere in the web interface. Fixes #1882

### CODE REVIEW

File: app/views/admin/users/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Create user" -%>
```
Comment (dsander) [inline app/views/admin/users/new.html.erb]: We should keep the capitalization consistent, I think I would prefer `Create User`.

File: app/views/agents/new.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "New agent" -%>
```
Comment (dsander) [inline app/views/agents/new.html.erb]: How about `Create Agent` to have the title state what action can be made on the page?

File: app/views/user_credentials/new.html.erb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "New Credential" -%>
```
Comment (dsander) [inline app/views/user_credentials/new.html.erb]: `Create Credential`?

File: app/views/devise/passwords/new.html.erb [hunk 4]
```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Forgot password" -%>
```
Comment (dsander) [inline app/views/devise/passwords/new.html.erb]: Maybe `Request password reset` or something similar?

Comment (strugee) [inline app/views/devise/passwords/new.html.erb]: I thought about that but I think it's too long. Remember that ` - Huginn` is appended to each title, so we should try to keep them as short as possible.

**Comment by strugee** [comment]
@dsander no problem! Happy to help!

Pushed adjustments to all the titles you mentioned except for the password reset one.

File: app/views/devise/passwords/new.html.erb (hunk 4 above)
Comment (dsander) [inline app/views/devise/passwords/new.html.erb]: Right, it pretty long how about just `Password reset`? I'd like to have the title match the action of the page.
We might also need to trim `Resend unlock instructions`.

Comment (strugee) [inline app/views/devise/passwords/new.html.erb]: @dsander yeah, I like that one. Pushed.

Re: `Resend unlock instructions `, perhaps s/instructions/code/ and call it good?

Comment (dsander) [inline app/views/devise/passwords/new.html.erb]: Sure, why not :)

Comment (strugee) [inline app/views/devise/passwords/new.html.erb]: @dsander pushed.

**Comment by cantino** [comment]
Odd that docker still shows as building, it looks like it passed when I view the build. I'm going to go ahead and merge. Thanks @strugee!

**Pull request merged**